### PR TITLE
Firebase auth

### PR DESCRIPTION
**Pull request by JimmyMcBride**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by AlyssaSaez — COMMENTED** [review]
Alert alert .env visible Dx

File: .env [hunk 1]
```diff
@@ -4,4 +4,5 @@ REACT_APP_FIREBASE_DOMAIN="getnutrijournal.firebaseapp.com"
 REACT_APP_FIREBASE_DATABASE="https://getnutrijournal.firebaseio.com"
 REACT_APP_FIREBASE_PROJECT_ID="getnutrijournal"
 REACT_APP_FIREBASE_STORAGE_BUCKET="getnutrijournal.appspot.com"
```
Comment (AlyssaSaez) [inline .env]: MEEP

**Review by AlyssaSaez — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (JimmyMcBride) [reply .env]: Done. :ok_hand:

**Review by AlyssaSaez — APPROVED** [review]
much better 😄

**Pull request merged**